expat/lib/xmlparse.c

```diff
@@ -2038,6 +2038,12 @@ XML_ParseBuffer(XML_Parser parser, int len, int isFinal) {
 
   if (parser == NULL)
     return XML_STATUS_ERROR;
+
```

Comment:
It looks like CVE-2024-45491 changes are missing ?
https://github.com/libexpat/libexpat/pull/891/files